src/aead/quic.rs

```diff
@@ -170,7 +170,7 @@ pub static CHACHA20: Algorithm = Algorithm {
 };
 
 fn chacha20_init(key: &[u8], _todo: cpu::Features) -> Result<KeyInner, error::Unspecified> {
-    let chacha20_key: &[u8; chacha::KEY_LEN] = key.try_into()?;
+    let chacha20_key: [u8; chacha::KEY_LEN] = key.try_into()?;
```

Comment:
Is removing the reference here to consume (rather than copy) the converted slice?

Reply:
> Is removing the reference here to consume (rather than copy) the converted slice?

Yes, I changed the key constructor to take the key by (moved) value, so I had to change this too.